develop/tutorials/articles/260-themes-and-layout-templates/xx-legacy-ant-theme-tasks.markdown

```diff
@@ -0,0 +1,188 @@
+# Using Legacy Ant Theme Tasks [](id=using-legacy-ant-theme-tasks)
+
+There are several ways you can go about developing a theme for Liferay. To help
+automate the theme development process, Liferay offers some build tools.
+
+Since Liferay 7.0 and Liferay DXP, you can create themes using the 
+[Themes Generator](/develop/tutorials/-/knowledge_base/7-0/themes-generator). 
+The Themes generator is a node based development tool that allows you to quickly 
```

Comment:
It does appear to be written as `node` in some of the other tutorials as well. I'll make a note to do a batch edit and change these to `Node.js`, so there is no confusion. thx!